framework/web/ContentRangeHttpException.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * @link http://www.yiiframework.com/
+ * @copyright Copyright (c) 2008 Yii Software LLC
+ * @license http://www.yiiframework.com/license/
+ */
+
+namespace yii\web;
+
+/**
+ * ContentRangeHttpException represents an exception caused by an improper request of the end-user.
+ * This exception uses when the requested range is not satisfiable.
+ *
+ * Throwing an ContentRangeHttpException like in the following example will result in the 416 page to be displayed.
+ *
+ * @author Залатов Александр <CaHbKa.Z@gmail.com>
+ */
+class ContentRangeHttpException extends HttpException
+{
+    /**
+     * Constructor.
+     * @param integer $status HTTP status code, default is 416.
+     * @param string $message error message
+     * @param integer $code error code
+     * @param \Exception $previous The previous exception used for the exception chaining.
+     */
+    public function __construct($status = 416, $message = 'Requested range not satisfiable', $code = 0, \Exception $previous = null)
```

Comment:
code should be hardcoded as it is the case with other Http exceptions: https://github.com/yiisoft/yii2/blob/af0119a910b9d6edd81d55d0b9395f27192edb2f/framework/web/NotFoundHttpException.php#L24